public/pages/CreateMonitor/components/AnomalyDetectors/AnomaliesChart/AnomaliesChart.js

```diff
@@ -62,9 +60,9 @@ const AnomaliesChart = props => {
                     showLegendDisplayValue={false}
                   />
                 ) : null}
-                <Axis id={getAxisId('bottom')} position="bottom" tickFormat={timeFormatter} />
```

Comment:
What was reasoning for this change?

Reply:
You are too quickly for me to get this PR truly ready, haha.

Reply:
I think it is a breaking change of Elastic Charts 18.0.0 https://github.com/elastic/elastic-charts/blob/a79e899a8d194e5559c184ca4f20920baef751bb/CHANGELOG.md#breaking-changes-1, and the sample change can be found in the link in my comment above.